### PR TITLE
Documentation Standard Updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This pull request was created automatically 🎉:

- [ ] Ensure `.github/` files have been properly updated (templates are updated and auto releaser is included)
- [ ] Confirm all appropriate end models are included in the `.quickstart/quickstart.yml` file as `public_models` (only for transform repos)
- Verify the README has made all necessary updates:
    - [ ] Removal of emojis
    - [ ] Replacing exclamation points with periods in sentences
    - [ ] Adjusting all headers to ensure only one H1 heading
    - [ ] Confirming a section header is included for Quickstart UI rendering (only for transform repos)
    - [ ] Renaming `**model` to `**table**` (only for transform repos)
    - [ ] Removal of Fivetran community and office hour resources (if necessary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,35 +11,35 @@
...
+- Enhances the task, users, projects, teams, and tags tables. Each of these tables is enriched with metrics that reflect the volume and breadth of current work and also the velocity of completed work.
 - Provides a daily metrics table, which lays out a timeline of task creations and completions to show the overall pace of deliverables.
 - Generates a comprehensive data dictionary of your source and modeled Asana data through the [dbt docs site](https://fivetran.github.io/dbt_asana/).
 - These tables are designed to work simultaneously with our [Asana source package](https://github.com/fivetran/dbt_asana_source).
 
-The following table provides a detailed list of all models materialized within this package by default. 
+The following table provides a detailed list of all models materialized within this package by default.
 > TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_asana/#!/overview?g_v=1&g_e=seeds).
 
 | **Model**                | **Description**                                                                                                                                |
```
Comment (fivetran-joemarkiewicz) [inline README.md:26]:
```suggestion
| **Todel**                | **Description**                                                                                                                                |
```

File: README.md [hunk 2]
```diff
@@ -11,35 +11,35 @@
...
 
-- Enhances the task, users, projects, teams, and tags tables. Each of these tables is enriched with metrics that reflect the volume and breadth of current work and also the velocity of completed work. 
+- Enhances the task, users, projects, teams, and tags tables. Each of these tables is enriched with metrics that reflect the volume and breadth of current work and also the velocity of completed work.
 - Provides a daily metrics table, which lays out a timeline of task creations and completions to show the overall pace of deliverables.
 - Generates a comprehensive data dictionary of your source and modeled Asana data through the [dbt docs site](https://fivetran.github.io/dbt_asana/).
 - These tables are designed to work simultaneously with our [Asana source package](https://github.com/fivetran/dbt_asana_source).
 
-The following table provides a detailed list of all models materialized within this package by default. 
+The following table provides a detailed list of all models materialized within this package by default.
```
Comment (fivetran-joemarkiewicz) [inline README.md:23]:
```suggestion

<!--section=“asana_transformation_model"-->
The following table provides a detailed list of all tables materialized within this package by default.
```

File: README.md [hunk 3]
```diff
@@ -11,35 +11,35 @@
...
 | [asana_project](https://fivetran.github.io/dbt_asana/#!/model/model.asana.asana__project)          | Each record represents an Asana project, enriched with metrics about their completed tasks, open tasks, and the users involved in the project. Also includes data about the project's most recently completed task and next due tasks. |
 | [asana__team](https://fivetran.github.io/dbt_asana/#!/model/model.asana.asana__team)             | Each record represents an Asana team, enriched with data about their completed tasks, open tasks, their projects, and the users involved with the team. |
 | [asana__tag](https://fivetran.github.io/dbt_asana/#!/model/model.asana.asana__tag)              | Each record represents an Asana tag, enriched with metrics about open and completed tasks associated with the tag. |
 | [asana__daily_metrics](https://fivetran.github.io/dbt_asana/#!/model/model.asana.asana__daily_metrics)    | Each record represents a single day, enriched with metrics about tasks opened at created that day. |
```
Comment (fivetran-joemarkiewicz) [inline README.md:34]:
```suggestion
<!--section-end-->
```

**Review by fivetran-avinash — APPROVED** [review]
Added GH_TOKEN, removed office hours, and  a small model-table tweak and we're good to go.

**Pull request merged**